pkg/api/v1/types.go

```diff
@@ -3404,6 +3404,15 @@ type PodExecOptions struct {
 	Command []string `json:"command" protobuf:"bytes,6,rep,name=command"`
 }
 
+// PodPortForwardOptions is the query options to a Pod's port forward call.
```

Comment:
@smarterclayton PTAL. I added more context so hopefully this will work better. The protocol version is the same for both SPDY and WebSockets since the encoding/decoding of the streams are identical. The only difference is how the streams are created and acknowledged, query param vs header.